### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by shatanikmahanty** [OpenCodeyard/ocyclient#4]

**Comment by Adelakin-Adewumi** [comment]
Hi, in case you are ready to add the description. i'd love to help with the documentation

**Comment by shatanikmahanty** [comment]
> Hi, in case you are ready to add the description. i'd love to help with the documentation

Definitely, we are finalizing some minor things about documentation. We will be finalizing by 2nd October. Happy contributing

**Comment by Adelakin-Adewumi** [comment]
> > Hi, in case you are ready to add the description. i'd love to help with the documentation
> 
> Definitely, we are finalizing some minor things about documentation. We will be finalizing by 2nd October. Happy contributing

Thank you

**Comment by shatanikmahanty** [comment]
@Adelakin-Adewumi I have updated the description. If you want to contribute to this issue, please review and get back to me.

**Comment by Adelakin-Adewumi** [comment]
> @Adelakin-Adewumi I have updated the description. If you want to contribute to this issue, please review and get back to me.

Alright, I will check it and get back to you tomorrow. Thank you.

**Comment by Adelakin-Adewumi** [comment]
> > @Adelakin-Adewumi I have updated the description. If you want to contribute to this issue, please review and get back to me.
> 
@shatanikmahanty I'd love to add my knowledge in the documentation if you don't mind. I will contribute the little I can in the readme file if this is okay by you.

**Comment by shatanikmahanty** [comment]
@Adelakin-Adewumi Go ahead. We will review your PRs and provide feedback so that you make changes if needed

**Comment by shatanikmahanty** [comment]
@Adelakin-Adewumi just checking in if you are working on this. Please reply to this comment to mark your participation. Reach out to us on community channels if you face issue setting up the project

**Comment by Adelakin-Adewumi** [comment]
@shatanikmahanty yes, I am. I will definitely reach out if I have an issue. Thank you for checking.

**Comment by shatanikmahanty** [comment]
@Adelakin-Adewumi unassigned you as there was no activity in the past few months! We look forward to your future contributions! In the mean time if you want any help or want to work on this issue again feel free to ask here!
